acceptance/helpers.rb

```diff
@@ -0,0 +1,130 @@
+require 'beaker-puppet'
```

Comment:
The `run_setup` and `run_teardown` helpers in this file are meant to mimic the setup and teardown functions from the original acceptance test helpers:

- `run_setup` mimics [`setup_puppet_on`](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/master/spec/spec_helper_acceptance.rb#L132)
	- The `use_master` keyword mimics the `agent: true` behavior from the original helper. I did not include an mcollective option, since release 2.0 of this moudule won't support puppet 3, where this was used before
- `run_teardown` mimcs [`teardown_puppet_on`](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/master/spec/spec_helper_acceptance.rb#L202)

I've put the minimal amount of functionality here; they will almost certainly need updates in the future. For example, I haven't included (or thoroughly tested) any instances where the `use_master` keyword is set, since I can only find two test cases (I think) in the existing test suite that use the equivalent functionality.